tests/test_create_storage_class_same_name.py

```diff
@@ -0,0 +1,67 @@
+"""
+A test for confirming that creation of two Storageclasses
+with the same name fails
+TC Name = OCS-322
+"""
+import os
+import yaml
+import logging
+import ocs.defaults as defaults
+
+from ocs import ocp
+from ocs import exceptions
+from utility import templating, utils
+from ocsci.enums import TestStatus
+
+log = logging.getLogger(__name__)
+TEMPLATE_DIR = "templates/ocs-deployment"
+TEMP_YAML_FILE = 'test_storageclass.yaml'
+SC = ocp.OCP(
+    kind='StorageClass', namespace=defaults.ROOK_CLUSTER_NAMESPACE
+)
+
+
+def create_storageclass(expect_fail=False, **kwargs):
+    """
+
+    Function to create a storage class and check for
+    duplicate storage class name
+
+    Args:
+        template_name (str): storageclass.yaml
+        expect_fail (bool): To catch the incorrect scenario in OC in case
+                            two SCs are indeed created with same name
+
+    """
+    SC_YAML = os.path.join(TEMPLATE_DIR, "storageclass.yaml")
+
+    log.info(f'creating yaml from template')
+    sc_yaml_file = templating.generate_yaml_from_jinja2_template_with_data(
+        SC_YAML, **kwargs)
+    with open(TEMP_YAML_FILE, 'w') as yaml_file:
+        yaml.dump(sc_yaml_file, yaml_file, default_flow_style=False)
+
+    log.info(f"Creating a new Storage Class with name provided by user ")
```

Comment:
Please print the class name kwargs['storageclass_name']

Reply:
done.